Repository/AbstractPhpcrRepository.php

```diff
@@ -147,6 +148,15 @@ public function add($path, $resource)
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function reorder($sourcePath, $position)
```

Comment:
I think using the index would be easier for frontend and backend libraries (correct me if I am wrong). Adding the index to the payload data would be a good convenience, but the FE library could infer it pretty easily